airflow/dag_processing/manager.py

```diff
@@ -572,6 +583,7 @@ def _read_from_direct_scheduler_conn(self, conn: MultiprocessingConnection) -> b
 
     def _refresh_requested_filelocs(self) -> None:
         """Refresh filepaths from dag dir as requested by users via APIs."""
+        return  # TODO: AIP-66 make bundle aware!
```

Comment:
this todo should have more info about what it means.  i don't know what you're talking about, and you might forget :)

Reply:
Done.